docs/manuals/ZAC-gebruikershandleiding/ZAC-gebruikershandleiding.md

```diff
@@ -284,33 +288,36 @@ Vanuit de werklijst kan met de productaanvraag via de ‘Zaak aanmaken’ functi
 
 Een productaanvraag verwijderen kan via het prullenbak icoon.
 
+<div style="page-break-after: always"></div>
+
 # Werk verdelen
 
 Vanuit de Zaken-werkvoorraad en Taken-werkvoorraad kunnen items verdeeld worden naar een groep en/of naar een behandelaar. Op deze manier kan een Coördinator de werkvoorraad managen en het werk verdelen binnen de organisatie.
 
 ## Werking van verdelen
 
-Als de Zaken-werkvoorraad of Taken-werkvoorraad geopend is dan kan de verdelen functie geactiveerd worden door een of meerdere van de items te selecteren. Met de Verdelen knop kan aangegeven worden aan welke groep en/of behandelaar de gekozen items toegekend moeten worden.  
+Als de Zaken-werkvoorraad of Taken-werkvoorraad geopend is dan kan de verdeelfunctie geactiveerd worden door een of meerdere van de items te selecteren. Met de Verdeel knop kan aangegeven worden aan welke groep en/of behandelaar de gekozen items toegekend moeten worden.  
 
 **Stappen:**
 
-1\. Open de Zaken-werkvoorraad of Taken-werkvoorraad
+1 Open de Zaken-werkvoorraad of Taken-werkvoorraad
 
-2\. Selecteer minimaal één item om te verdelen door deze links aan te vinken
+2 Selecteer minimaal één item om te verdelen door deze links aan te vinken
 ![Selecteer minimaal een item](./images/selecteer-minimaal-een-item.png)
 
-3\. Klik op de Verdelen onderaan het scherm
+3 Klik op 'Verdelen' onderaan het scherm
 
-4\. Kies de groep en/of de behandelaar aan wie de items toegekend moeten worden. Er kan alleen een behandelaar gekozen worden die bij de gekozen groep hoort.
+4 Kies de groep en/of de behandelaar aan wie de items toegekend moeten worden. Er kan alleen een behandelaar gekozen worden die bij de gekozen groep hoort.
 ![Kies groep en / of behandelaar](./images/kies-groep-en-of-behandelaar.png)
 
-5\. Vul eventueel een reden in, deze wordt als toelichting gebruikt in de historie van de zaak
+5 Vul eventueel een reden in, deze wordt als toelichting gebruikt in de historie van de zaak
 
-6\. Klik op ‘Verdelen’ om de actie uit te voeren
+6 Klik op ‘Verdelen’ om de actie uit te voeren
+> Let op! <br>Als eenmaal 'Verdelen' is gekozen, is dit niet meer te onderbreken.
```

Comment:
hi @camielb , ik zou zelf kiezen voor 'gestart' ipv 'gekozen'. Want wanneer heb je precies gekozen in deze flow?

Reply:
Je hebt de keuze tussen Verdelen of annuleren, vandaar dacht ik aan kiezen